# UMP11: Optimierung von Marketingausgaben

## Lösung LP in Matrix Notation
Nachfolgendes Skript zeigt die Lösung des Modells als LP, d.h. als lineares Programm, ausgehend von einer Formulierung in **Matrix-Notation**. 


In [1]:
from UMP11_LP_MATRIX_NOTATION import *

main()

Optimization terminated successfully. (HiGHS Status 7: Optimal)
Objective value:	167708.33333333334
Results:			[116.66666667 320.83333333]


Wir haben hier das Problem als LP formuliert, obwohl die Anzahl Werbespots natürlich Ganzzahligkeit fordern. Hier ist die Bedingung aber gegeben, das Problem als LP zu formulieren und auf geeignete Weise zu runden. Damit kommen wir zu folgender Lösung:

Die Parkleitung muss 117 Spots unter der Woche schalten und 321 Spots am Wochenende. Die Werbekosten belaufen sich damit auf 167'850 CHF.